Add ac97 alias

libxl changed from supplying "-soundhw" to "-device" in commit
e85e2a3c17b6 "tools/libxl: Replace deprecated -soundhw on QEMU command
line".  However, QEMU expects the uppercase "-device AC97" whereas libxl
supplies lowercase "-device ac97", which matches the old -soundhw
option.  QEMU ~5.1 added a compatibility alias in commit 2957f5ada751
"audio: deprecate -soundhw ac97".  Backport a minimal compatibility
alias.

Without the alias, qemu fails to start with:
-device ac97: 'ac97' is not a valid device model name

--- a/qdev-monitor.c
+++ b/qdev-monitor.c
@@ -48,6 +48,7 @@ typedef struct QDevAlias
 
 /* Please keep this table sorted by typename. */
 static const QDevAlias qdev_alias_table[] = {
+    { "AC97", "ac97" }, /* -soundhw name */
     { "e1000", "e1000-82540em" },
     { "ich9-ahci", "ahci" },
     { "lsi53c895a", "lsi" },
